xtask/src/commands/release.rs

```diff
@@ -497,6 +503,13 @@ impl Prepare {
     ///   (If not installed, you should [install `helm-docs`](https://github.com/norwoodj/helm-docs))
     fn update_helm_charts(&self, version: &str) -> Result<()> {
         println!("updating helm charts");
+
+        replace_in_file!(
+            "./helm/chart/router/Chart.yaml",
+            "version: \"v[^\"]+\"",
+            format!("version: \"{}\"", version)
```

Comment:
Hmm, this doesn't and won't match the current pattern of what is in the `Chart.yaml` file because the:

1. Version doesn't use the `v` prefix.
2. Isn't quoted.

I think it's fine if we _start_ quoting it, but choose one of these two suggestions, I think?

### Quoted

```suggestion
            "^version:.*?$",
            format!("version: \"{}\"", version)
```

### Unquoted

```suggestion
            "^version:.*?$",
            format!("version: {}", version)
```

Probably worth testing though as I just wrote these into GitHub Suggestions.

Reply:
Fixed in [c576037](https://github.com/apollographql/router/pull/2381/commits/c5760370b614531ffe6b7b72df4733f229308d7f) + manually tested